### PR TITLE
[Common Lisp] Add 'implementing concept exercise' document.

### PR DESCRIPTION
**Pull request by verdammelt**

### CODE REVIEW

**Comment by verdammelt** [comment]
@SleeplessByte whoops! and thanks for the rename.

File: languages/common-lisp/docs/implementing-a-concept-exercise.md [hunk 1]
```diff
@@ -1,5 +1,89 @@
...
+done this, please read the [Common Lisp concept exercises
+introduction][concept-exercises].
+
+To implement a concept exercise, the following files must be created:
+
+<pre>
+languages
+└── csharp
+    └── exercises
+        └── concept
```
Comment (TheLostLambda) [inline languages/common-lisp/docs/implementing-a-concept-exercise.md]: I think that a lonely `csharp` got left behind here. It should probably be changed to `common-lisp` like it is in v2.

Comment (verdammelt) [reply languages/common-lisp/docs/implementing-a-concept-exercise.md]: Good catch. I replaced C# but didn't even see that csharp.

**Comment by verdammelt** [comment]
Fixed the `csharp` and also removed references to `.meta/config.json` (ref: #341, #360, #362)

**Pull request merged**